### PR TITLE
F/667-Unify close buttons and design

### LINKED ISSUE #667: Unify close buttons
**Is your feature request related to a problem? Please describe:**  
We use different close buttons/options which can be confusing for the user. 

**Please describe the solution you'd like:**  
For more intuitive work we should use the same close buttons everywhere. It won't replace the current close function (pressing the same button will open and close the window), just adds another close option which we already use for other dialogues (e.g. Workload Generation). 

**Affected Component(s):**  
`Frontend`, `UI`

Before: 
Screenshot 2020-07-13 at 18 41 49

Changes:
- Added close button (X) 
- Adapted Design (colors) to other dialogues e.g. Workload Generation (same font, size, color etc.)
- Remove Caps Lock for consistent typography
- Changed title from "Comparison" to "Comparison Page Settings". 
- Changed title "Data" to Databases and Metrics

Screenshot 2020-07-14 at 10 36 43

Screenshot 2020-07-14 at 10 39 24

Changes: 
- Inserted Remove Icon into DB Button 
- Added "SQL" for better understanding of the Icon
- Added close button (X) 
- Adapted Design (colors) (font, size, color etc.)

Screenshot 2020-07-14 at 16 10 35

More changes (from left to right): 
Screenshot 2020-07-18 at 10 17 26
Screenshot 2020-07-18 at 10 17 34Screenshot 2020-07-18 at 11 53 58

Screenshot 2020-07-18 at 12 16 06Screenshot 2020-07-18 at 12 12 08

### PR DESCRIPTION
**Pull request by monasobh**
# Resolves #667 

**Does your pull request solve a problem? Please describe:**  
We now have a similar close button and similar design on the different windows/dialogues. 

**Does your pull request add a feature? Please describe:**  
See pictures in Isssue #667 

**Affected Component(s):**  
`Settings`, `UI`,

### CODE REVIEW

File: hyrisecockpit/frontend/src/components/selection/DatabaseSelection.vue [hunk 1]
```diff
@@ -93,3 +93,10 @@ export default defineComponent({
   },
 });
 </script>
+<style scoped>
+.v-card__subtitle,
+.v-card__text,
+.v-card__title {
+  padding: 6px;
+}
```
Comment (schTi) [inline hyrisecockpit/frontend/src/components/selection/DatabaseSelection.vue]: Its no good style to directly overwrite vuetify classes. As you have access to all of those elements, just add a simple custom class and use it for subtitle, title, and text of the card

File: hyrisecockpit/frontend/src/components/selection/MetricSelection.vue [hunk 2]
```diff
@@ -95,3 +95,10 @@ export default defineComponent({
   },
 });
 </script>
+<style scoped>
+.v-card__subtitle,
+.v-card__text,
+.v-card__title {
+  padding: 6px;
+}
+</style>
```
Comment (schTi) [inline hyrisecockpit/frontend/src/components/selection/MetricSelection.vue]: same here

File: hyrisecockpit/frontend/src/components/selection/SelectionList.vue [hunk 3]
```diff
@@ -437,5 +449,17 @@ function useStaticRangeSelection(
   display: block;
   margin-left: auto !important;
   margin-right: auto !important;
+  font-size: medium;
+}
+.v-tab {
+  height: 48px;
+  text-transform: none !important;
```
Comment (schTi) [inline hyrisecockpit/frontend/src/components/selection/SelectionList.vue]: same for v-tab here, you could just use a custom class

File: hyrisecockpit/frontend/src/components/selection/SelectionList.vue [hunk 4]
```diff
@@ -437,5 +449,17 @@ function useStaticRangeSelection(
...
+}
+.button {
+  text-transform: none !important;
```
Comment (schTi) [inline hyrisecockpit/frontend/src/components/selection/SelectionList.vue:457]: whats the reason for this?

File: hyrisecockpit/frontend/src/components/selection/SelectionList.vue [hunk 5]
```diff
@@ -213,6 +221,10 @@ export default defineComponent({
       type: Boolean,
       default: false,
     },
+    onClose: {
+      type: Function,
+      default: null,
+    },
```
Comment (schTi) [inline hyrisecockpit/frontend/src/components/selection/SelectionList.vue]: Looks like React-style 😄 . In Vue you would normally emit a `close`-event to the parent, and the parent would call the `onClose`- function on his own. So you dont need to pass the closing function to the child

**Review by schTi — CHANGES_REQUESTED** [review]

File: hyrisecockpit/frontend/src/components/selection/SelectionList.vue (hunk 5 above)
Comment (monasobh) [reply hyrisecockpit/frontend/src/components/selection/SelectionList.vue]: Ok. That's also how the Plugins close button works.. Maybe we should find a consistent solution 👍

File: hyrisecockpit/frontend/src/components/selection/SelectionList.vue (hunk 4 above)
Comment (monasobh) [reply hyrisecockpit/frontend/src/components/selection/SelectionList.vue:457]: it allows the text in a button to be in small letters not caps lock

File: hyrisecockpit/frontend/src/components/selection/SelectionList.vue (hunk 5 above)
Comment (schTi) [reply hyrisecockpit/frontend/src/components/selection/SelectionList.vue]: Yes, because Max implemented this :D. Consistent would be to use events only

**Review by schTi — COMMENTED** [review]
Please resolve those conflicts before you continue, this will apply some changes to the test code as well

**Comment by schTi** [comment]
You removed the ID of the add database button. To fix the tests, just add `data-id="add-database-button"` to the new button

**Review by schTi — APPROVED** [review]
Great work!

**Comment by monasobh** [comment]
> Great work!

Did some more changes. Please re-review. Thanks!

**Review by schTi — APPROVED** [review]

**Pull request merged**